packages/framework/store/src/yjs/awareness.ts

```diff
@@ -9,14 +9,16 @@ export interface UserInfo {
   name: string;
 }
 
+type UserSelection = Array<Record<string, unknown>>;
+
 // Raw JSON state in awareness CRDT
 export type RawAwarenessState<
   Flags extends Record<string, unknown> = BlockSuiteFlags,
 > = {
   user?: UserInfo;
   color?: string;
   flags: Flags;
-  selection: Array<Record<string, unknown>>;
+  selection: Record<string, UserSelection>;
```

Comment:
Could this change cause the old version to crash? It is better to use `v2` field?

Reply:
Done